dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPI.java

```diff
@@ -77,7 +77,7 @@ public List<FileAsset> findFileAssetsByParentable(final Parentable parent,
 	 * @return
 	 * @throws DotStateException
 	 */
-	List<IFileAsset> fromContentletsI(List<Contentlet> cons) throws DotStateException;
+	List<IFileAsset> fromContentletsI(List<Contentlet> cons) throws DotStateException, DotDataException, DotSecurityException;
```

Comment:
not really fan of breaking the backwards compatible, catch the new exception and wrap into DOtRuntime